### ISSUE TITLE
Allocate and register mariners viewing group for `INDHLT`

### ISSUE THREAD

**Issue by DavidGrant-NIWC** [S-101-Portrayal-subWG/Working-Documents#136]
S-52 does not specify a viewing group for `INDHLT`.

Recommend `53020` since `DNGHLT` is assigned to viewing group `53010`.

![image](https://user-images.githubusercontent.com/62906211/228937472-7170c6ef-cf17-438e-9ca9-b229422fb267.png)

**Comment by alvarosanuy** [comment]
**Agree with** @DavidGrant-NIWC . Allocating Viewing Group 53020 to INDHLT seems reasonable to me as well.

If approved to proceed by the PsWG:

- [ ] Needs to be added to S-98 - Table C-14 (maybe include action in https://github.com/S-101-Portrayal-subWG/Working-Documents/issues/135

- [ ] Needs to be added to the GI Registry. @alvarosanuy could do this now or add it to the list of VG that are still pending to be registered. Refer to https://github.com/S-101-Portrayal-subWG/Working-Documents/issues/131

**Comment by DavidGrant-NIWC** [comment]
TL;DR:
- [x] The viewing groups listed below are used in the 1.1.1 PC.
- [ ] Request approval of the new viewing groups and viewing group (sub-)layer.
- [ ] Register the viewing groups (and update `53010`) on approval.

---
This is a little more complicated then I initially thought. Further investigation reveals that `INDHLT` is assigned to `53010` in the S-52 lookup tables. However, there is a conflicting requirement:
![image](https://user-images.githubusercontent.com/62906211/229154383-fcee4a62-b532-46db-aef6-6e65ee9b2292.png)

To support this requirement we need to provide separate viewing groups for each type of prohibited area or area with special conditions. See the table below.

The PC also adds a new viewing group (sub) layer (which should be registered) under `10 - Miscellaneous (Standard)` to group all the alert highlights together:
![image](https://user-images.githubusercontent.com/62906211/229162805-2ff02c17-0758-41d6-bd5d-b5c94e788200.png)

If desired, we could add additional sub-layers to `Alert Highlights (Standard)`: `Safety Contour`, `Navigational Hazards`, and `Prohibited Areas`; the PC does not currently provide these additional sub-layers.

---
### Proposed viewing groups for alert highlights
|Viewing Group|Name|Description|
|:---|:---|:---|
|`53010`|Safety Contour Highlight|Used to turn on or off the graphical highlighting of alerts triggered by the safety contour.|
|`53011`|Navigational Hazards Highlight|Used to turn on or off the graphical highlighting of alerts triggered by navigational hazards|
|`53012`|Traffic Separation Zone Highlight|Alert highlight: Traffic Seperation Zone|
|`53013`|Inshore Traffic Zone Highlight|Alert highlight: Inshore Traffic Zone|
|`53014`|Restricted Area Highlight|Alert highlight: Restricted Area|
|`53015`|Areas to be Avoided Highlight|Alert highlight: Areas to be avoided|
|`53016`|PSSA (Particularly Sensitive Sea Area) Highlight|Alert highlight: PSSA (Particularly Sensitive Sea Area)|
|`53017`|Caution Area Highlight|Alert highlight: Caution area|
|`53018`|Offshore Production Area Highlight|Alert highlight: Offshore production area|
|`53019`|User Defined Areas to be Avoided Highlight|Alert highlight: User defined areas to be avoided|
|`53020`|Military Practice Area Highlight|Alert highlight: Military practice area|
|`53021`|Seaplane Landing Area Highlight|Alert highlight: Seaplane landing area|
|`53022`|Submarine Transit Lane Highlight|Alert highlight: Submarine transit lane|
|`53023`|Anchorage Area Highlight|Alert highlight: Anchorage area|
|`53024`|Marine Farm/Aquaculture Highlight|Alert highlight: Marine farm/aquaculture|

Only `53010` is currently registered (it has a duplicate entry, and the name and definition should be updated).

---
### Proposed mapping of alert highlights to viewing groups (see S-52 PresLib 4.0.3 clause 10.5.9, 10.5.10 and 10.5.12)
|Highlight|Feature Type|Condition (if any)|Geometry|Viewing Group|
|:---:|:---:|:---:|:---:|:---:|
|Safety Contour|Multiple|||`53010`|
|Navigational Hazards|Multiple|||`53011`|
|Prohibited Area/Special Conditions|`SeparationZoneOrLine`||Surface|`53012`|
||`InshoreTrafficZone`|||`53013`|
|Prohibited Area/Special Conditions|`RestrictedAreaNavigational`|`categoryOfRestriction` is not `14` or `28`||`53014`|
|Prohibited Area/Special Conditions|`RestrictedAreaRegulatory`|`categoryOfRestriction` is not `14` or `28`||`53014`|
|Prohibited Area/Special Conditions|`RestrictedAreaNavigational`|`categoryOfRestriction` = `14`||`53015`|
|Prohibited Area/Special Conditions|`RestrictedAreaRegulatory`|`categoryOfRestriction` = `14`||`53015`|
|Prohibited Area/Special Conditions|`RestrictedAreaNavigational`|`categoryOfRestriction` = `28`||`53016`|
|Prohibited Area/Special Conditions|`RestrictedAreaRegulatory`|`categoryOfRestriction` = `28`||`53016`|
|Prohibited Area/Special Conditions|`CautionArea`|||`53017`|
|Prohibited Area/Special Conditions|`OffshoreProductionArea`|||`53018`|
|Prohibited Area/Special Conditions|User defined areas to be avoided|||`53019`|
|Prohibited Area/Special Conditions|`MilitaryPracticeArea`|||`53020`|
|Prohibited Area/Special Conditions|`SeaplaneLandingArea`|||`53021`|
|Prohibited Area/Special Conditions|`SubmarineTransitLane`|||`53022`|
|Prohibited Area/Special Conditions|`AnchorageArea`|||`53023`|
|Prohibited Area/Special Conditions|`MarineFarmCulture`|||`53024`|

**Comment by DavidGrant-NIWC** [comment]
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/44a2427e-e892-4963-b786-63142de37188)

**Comment by alvarosanuy** [comment]
**Decisions made at Portrayal subWG meeting on 10/5/23**

1. Approve NIWCs mapping and VG allocations for INDHLT. INDHLT can't be allocated to just one VG (i.e 53020 as initially proposed) due to a requirement in (**document requested to NIWC below**) where highlights on features in the 'Areas for which special conditions exist' group need to be able to be turned On/Off individually (by feature/attribute combination). 
**NIWC**  - Can you please let me know where did you get this extract from? Is it IEC 61174?? Shouldn't be an S-64 test for this if it's an existing requirement??
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/70189016/b7481204-7e9d-4355-83fb-ee5afc3a8984)

- [ ] 2. **NIWC** to prepare and submit S-98 change request as necessary. It may be easier to maintain if there's only one entry for INDHLT covering VGs 53011 to 53029. These layers may grow if new A&I proposals are approved for additional 'Special Areas'. The final authority will effectively become the entries in the S-101 PC and we should try to minimize misalignments with S-98.
- [ ] 3. **Alvaro** to register the new VGs and update VG 53010 entry in the Registry as required. 
- [x] 4. Request S-164 test.
- [x] 5. **Alvaro** to include this topic in his PsWG report to S101PT10. **The main thing to highlight** is that IEC 61174 requirement does not come from any other IMO or IHO instrument and there's no S-64 check to ensure it has been systematically implemented by OEMs. If this is the case, and we do implement more rigorously in S-101 A&I and include S-164 check, we run the risk that in DF-ECDIS, mariners will get different results depending on the product at use (S-57 or S-101). Switching OFF the Indications Highlights may turn off all graphical indications (DNGHLT and INDHLT) in S-52 (as both are linked to VG 53010) but would only turn off DNGHLT in an S-101 dataset. All yellow graphical alerts would continue active on the ECDIS screen.
- [x] The PsWG will seek endorsement to implement, in PC 1.2.0, the subdivision of Indication Highlights into numerous VGs for the feature/attribute combinations included in the Alert's groups 'Areas for which special conditions exist' as presented by NIWC.
- [ ] @DavidGrant-NIWC - You will need to remove all entries related to 'Restricted Area Regulatory' as per the decision made during our discussions in https://github.com/S-101-Portrayal-subWG/Working-Documents/issues/88. These features will not generate A&I.

**Comment by DavidGrant-NIWC** [comment]
> 1. **NIWC**  - Can you please let me know where did you get this extract from? Is it IEC 61174?? Shouldn't be an S-64 test for this if it's an existing requirement??
>    ![image](https://user-images.githubusercontent.com/70189016/239152067-b7481204-7e9d-4355-83fb-ee5afc3a8984.png)

Yes, it's a new requirement introduced in 61174 which doesn't seem to be derived from a PS requirement. See IEC 61174 ed 4.0 clause 4.10.2.1 (route planning) and 4.10.3 (route monitoring). If valid, it probably should be tested in S-64, however S-64 only tests a small subset of the requirements.

**Comment by alvarosanuy** [comment]
@DavidGrant-NIWC - Can you please provide an update on the following actions?
> * 2. **NIWC** to prepare and submit S-98 change request as necessary. It may be easier to maintain if there's only one entry for INDHLT covering VGs 53011 to 53029. These layers may grow if new A&I proposals are approved for additional 'Special Areas'. The final authority will effectively become the entries in the S-101 PC and we should try to minimize misalignments with S-98


The following action may require a change in the implementation strategy (by CATREA value instead of feature type) **if** the proposal to merge Restricted Areas Regulatory and Navigational back into one feature is approved by the S101PT (SHOM paper to S101PT11).

> * @DavidGrant-NIWC - You will need to remove all entries related to 'Restricted Area Regulatory' as per the decision made during our discussions in [Alerts & Indications #88](https://github.com/S-101-Portrayal-subWG/Working-Documents/issues/88). These features will not generate A&I.

**Comment by DavidGrant-NIWC** [comment]
> @DavidGrant-NIWC - Can you please provide an update on the following actions?
> 
> > * 2. **NIWC** to prepare and submit S-98 change request as necessary. It may be easier to maintain if there's only one entry for INDHLT covering VGs 53011 to 53029. These layers may grow if new A&I proposals are approved for additional 'Special Areas'. The final authority will effectively become the entries in the S-101 PC and we should try to minimize misalignments with S-98

I'm still unclear on how we should proceed. S-98 needs to be updated as it doesn't show any entry for indication highlight. Should we assign all the indication highlights to 53010 to match S-52? This will not allow meeting the 61174 requirement cited above, but I guess no one is meeting it currently anyway and the alert catalog could be updated in a future iteration to meet the requirement if desired.

So, if you agree we will:
* Submit an S-98 change proposal to add "Indication highlight (indhlt)" to viewing group 53010 in S-98 Table C-14 
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/4c2cdaaf-51c4-42bc-917a-ef67d62f0509)
* Update the PC to remove the various viewing groups for different indication highlights and assign them all to 53010.
* Update viewing group 53010 in the PC:

|attribute|Old|New|
|:---|:---|:---|
|name|Safety Contour Highlight|Alert Highlight|
|description|Used to turn on or off the graphical highlighting of alerts triggered by the safety contour|Used to turn on or off the graphical highlighting of alerts|

**Comment by alvarosanuy** [comment]
Hi Dave,

After my PsWG presentation to S101PT10 (June 2023), Hannu sent us an email showing that the requirement to enable granular selection of highlights by mariners in ECDIS is not only specified in IEC 61174 but also in IMO MSC.530(106).
He also pointed us to S-64 test 6.1. For me, the existence of this test shows that the functionality (selective highlight of areas for which special conditions exist) is currently a requirement for (S-57) Type approved ECDIS.

Also, in regards to the fact that S-52 and S-98 do not currently allocate a VGL to INDHLT, is it possible this was done on purpose to not restrict INDHL to one VGL knowing the IMO requirement?? In short, that it was left up to OEMs to split INDHLT into several layers? Irrespective of this, I believe that this functionality has to be standardized and spelled out in a new version of S-98 Annex C.
Furthermore, there's no evidence that switching off VGL 53010 will also switch off all INDHLTs. Based n S-64 test, it is more logical to think that all the INDGHLT implementation by OEMs was done using self-allocated VGL (or other mechanism).
My point here is that our concerns around different behaviors between S-52 and S-101 when switching off VGL 53010 is probably unwarranted (this concern was highlighted in my presentation to S101PT10). There's a high chance that when VGL 53010 is turned off in S-52, only DNGHLT is turned off (as indicated in the Table).

**Based on this, I'm of the opinion we need to proceed as decided by the Portrayal subWG at the 10/5/23 meeting**
This means:

- [ ] Retain the VGL subdivision currently implemented in PC 1.1.1 (listed above and endorsed by the PsWG). 
- [ ] Register VGL 53011 to 53024 - **Alvaro** (or KHOA/IHO as part of a bigger project to update all VG & VGL).
- [ ] **NIWC** to prepare and submit S-98 change request to include, in S-98 Annex C- Table C-14, VGL 53011 to 53024.
I think S-98 should also instruct OEMs to establish a unique selector for all INDHLT VGL (so they can all be selected/deselected at the same time - the default setting should be 'all ON'). Hanging from that, mariners must have access to the individual list of VGL to allow them to selectively switch on/off INDHLT by feature, as required by IMO MSC.530(106).
- [ ] **Alvaro** Update VGL 53010 in the GI Registry as follows:   -  (PC 1.1.0 should already read like this)
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/70189016/b4b0384b-e3f6-4f91-a590-e8431c4b9f34)

Note that an S-164 test to match S-64 test 6.1 should be created without request by the S-164 sWG, but I have added a Label to this issue anyway. Depending on how S-98 changes are finally processed this Test may require some fine tuning.

Not sure we should mention any of this in the PsWG report to S101PT11. My recollection of S101PT10 is that the team did not provided any comment on the subdivisions or potential discrepancies in the Alerts behaviour during DF-ECDIS period.
The only comment I recall was from Hannu and it was to challenge my statements that the requirement was only in IEC 611174 and that there was no S-64 check. The fact he was right, further reinforces the idea of moving ahead with implementation as per the actions listed above.

**Comment by DavidGrant-NIWC** [comment]
### Summary

@alvarosanuy agree with all you wrote, except that we may remove 53010 from S-98 instead of adding 53011 - 53024. We will work with Raphael to determine the appropriate action.

See https://github.com/iho-ohi/98-interoperability/issues/11

### Viewing group layer for INDHLT
> I think S-98 should also instruct OEMs to establish a unique selector for all INDHLT VGL (so they can all be selected/deselected at the same time - the default setting should be 'all ON').

I think this is already addressed - copied from my msg of MAR 31:
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/376c3247-a64b-4b68-ba6a-210dfa58e7c9)

Would you like us to break the Alert Highlights viewing group layer down further?

### Viewing groups
> Hanging from that, mariners must have access to the individual list of VGL to allow them to selectively switch on/off INDHLT by feature, as required by IMO MSC.530(106).

Agreed, they are already present in the current PC:
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/06bcfc6e-6fbe-492d-ba23-df0fea6e2547)

### S-52 assignment of INDHLT

> Also, in regards to the fact that S-52 and S-98 do not currently allocate a VGL to INDHLT, is it possible this was done on purpose to not restrict INDHL to one VGL knowing the IMO requirement?

You can see below that S-52 does assign INDHLT to 53010, it just isn't listed in the document:
|S-52 PresLib 4.0.3 Part 1|PresLib 4.0.3 lookup tables|
|:---|:---|
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/92aa3e3a-c59a-4bda-a28c-59b3299c7da2)|![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/4fa0aa1f-675b-470d-b31b-14d7ebe590f6)

S-98 Annex C just copied the values from the S-52 document, so that explains why it isn't present there.

**Comment by alvarosanuy** [comment]
> Would you like us to break the Alert Highlights viewing group layer down further?

I think it would very handy for mariners to have the option to deselect all the INDHLT and leave DNGHLT turned On. This option was not available in S-52 by having both highlights under one VGL.

> You can see below that S-52 does assign INDHLT to 53010, it just isn't listed in the document:

Sorry, I overlooked this bit. To clarify then:

- S-52 has one layer (53010) that turns DNGHLT and INDHLT on/off at the same time.
- S-101 PC & S-98 Annex C Table, will use:
               -  53010 for DNGHLT & 53011-53024 for INDHLT under one selector. This is your '_Alerts Highlights (Standard)_' layer in the on-shore ECDIS. This would deliver same performance than S-52. S-98 should request the existence of this layer so mariners can get the same behavior than in S-52.
               - A new ECDIS selector to independently switch ON/OFF DNGHLT (53010) and INDHLT (53011-53024) VGLs.
               Under your '_Alerts Highlights (Standard)_' layer ?? Is this possible??
               If implemented, I also recommend S-98 requests the existence of these individual selectors.

**Comment by DavidGrant-NIWC** [comment]
> A new ECDIS selector to independently switch ON/OFF DNGHLT (53010) and INDHLT (53011-53024) VGLs.
Under your 'Alerts Highlights (Standard)' layer ?? Is this possible??
If implemented, I also recommend S-98 requests the existence of these individual selectors.

I think this might make the most sense, since this is how the required indications are organized, and also matches the alert catalog entries:
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/3ee1e015-85ae-4c04-a4c8-fd0793629778)

|Disabled Highlight|Result|
|:---:|:---:|
|Safety Contour|![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/748e7105-7e3d-416d-8214-a272f5e24fdf)|
|Nav Hazards|![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/bf873cff-43ec-450b-bd81-ee3fa7e91c00)|
|Prohibted Areas|![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/99da74db-dd17-4eb9-b24b-79130f1203db)|

```
	  <viewingGroupLayer id="10b">      <!-- S-101 Alert Highlights -->
		<description>
            <name>Safety Contour Highlight (Standard)</name>
            <description>Toggles graphical highlighting of alerts triggered by the safety contour</description>
            <language>eng</language>
		</description>
		 <!-- Alert Highlights -->
		 <viewingGroup>53010</viewingGroup>
	  </viewingGroupLayer>
	  <viewingGroupLayer id="10c">      <!-- S-101 Alert Highlights -->
		<description>
            <name>Navigation Hazards Highlight (Standard)</name>
            <description>Toggles graphical highlighting of alerts triggered by navigational hazards</description>
            <language>eng</language>
		</description>
		 <!-- Alert Highlights -->
		 <viewingGroup>53011</viewingGroup>
	  </viewingGroupLayer>
	  <viewingGroupLayer id="10d">      <!-- S-101 Alert Highlights -->
		<description>
            <name>Prohibited Areas Highlight (Standard)</name>
            <description>Toggles graphical highlighting of alerts triggered by prohibited areas or areas with special conditions</description>
            <language>eng</language>
		</description>
		 <!-- Alert Highlights -->
		 <viewingGroup>53012</viewingGroup>
		 <viewingGroup>53013</viewingGroup>
		 <viewingGroup>53014</viewingGroup>
		 <viewingGroup>53015</viewingGroup>
		 <viewingGroup>53016</viewingGroup>
		 <viewingGroup>53017</viewingGroup>
		 <viewingGroup>53018</viewingGroup>
		 <viewingGroup>53019</viewingGroup>
		 <viewingGroup>53020</viewingGroup>
		 <viewingGroup>53021</viewingGroup>
		 <viewingGroup>53022</viewingGroup>
		 <viewingGroup>53023</viewingGroup>
		 <viewingGroup>53024</viewingGroup>
	  </viewingGroupLayer>
```

**Comment by alvarosanuy** [comment]
**Thanks Dave.** Please move ahead and coordinate S-98 Annex C changes with the S100WG.
We can leave this Issue open until S-98 changes are endorsed by S100WG and commit changes to PC 1.2.0 only then?

Do you have any other questions?

**Comment by DavidGrant-NIWC** [comment]
I think this group should be the one advancing proposals to the S-101PT for approval and incorporation into the S-101 PC. Requiring approval from the S-100WG for S-101 PC changes means that it will be increasingly difficult to get any changes made, and being able to modify the catalogs was a primary reason to develop S-100.

S-98 is only for interoperability and ECDIS requirements. I'm waiting on a response from @rmalyankar to https://github.com/iho-ohi/98-interoperability/issues/11 before proceeding with the S-98 change proposals. In my view S-98 Annex C should only document VG's/VGL's that aren't registered. If S-98 is going to provide a list of VG's/VGL's it should be informative and periodically updated from the portrayal registry.

That said, S-98 currently says:
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/93f3a1da-c380-468a-a148-826fbd3509d6)

**Comment by alvarosanuy** [comment]
@DavidGrant-NIWC - What are the key statements you think we should get endorsed by the PT? I can add that to my presentation to S101PT11.

On another note, I thought S-98 was the official place to specify VG and VGLs for ECDIS implementation.
What do you mean by: _In my view S-98 Annex C should only document VG's/VGL's that aren't registered._ Everything the S101PT considers important enough as to be standardised, is to be considered mandatory and registered in IHO's GI Registry. Other 'optional' elements would be up to implementers to manage their way. I believe all the layers and  groups we discussed above are to be mandatory and not optional.

Agree that, with the introduction of the GI Registry, some content that is currently provided in a range of publications will most certainly become redundant and create confusion and misalignment (more than one point of truth). One of the problems we face at the moment is that the GI Registry is lagging behind and is not the Point of Truth. Once this problem is resolved and the process to keep it up to date becomes efficient (timely updated), I agree some content listed in documents (i.e. S-98) could be discontinued and point to the GI Registry (or the relevant latest versions of catalogues?) instead.

**Comment by DavidGrant-NIWC** [comment]
> What are the key statements you think we should get endorsed by the PT? I can add that to my presentation to S101PT11.

Not something I've thought about - you may want to document/get endorsement for the working process, but maybe this has already been done? I'm more focused on the technical issues. I think the process is:

PSWG consensus -> PC implementation -> (repeat previous steps until) PSWG approval -> submit PC to PT for approval?

> On another note, I thought S-98 was the official place to specify VG and VGLs for ECDIS implementation.

Ironic that we advocate paper documents for a product which replaced a paper document. Anyway, I don't see the value in providing any of this information in Annex C. If the information is intended to be used by portrayal elements it should be in the registry, and you should be able to get a listing of the registered elements. S-98 Annex C could tell you how to do that or, as you recommend, point you to the registry.

Some argue for duplicating the registered information in publications such as S-98 Annex C, but in my mind (and as you point out):
* It creates potential for disagreements between the various sources of information (this is already happening).
* Documents are hard to update and hard to get approved - one of the main drivers for moving to S-100.
* Even if a process is developed to keep the document / registry / PC's aligned, using the document as the normative source will slow the iterative/Agile development processes.
* Viewing groups which are purely for use by the ECDIS (are unused by the portrayal catalog) don't really need a common implementation across ECDIS systems.
  * Does it really matter if system A uses "53040" for mariners notes and system B uses "154321"?
    * Maybe you could argue they should have a common name "Mariners Notes". In that case, the viewing group should be registered.
  * We just need to ensure that system implementers use viewing groups that don't conflict with registered viewing groups.
    * I imagine that OEM's will applying a prefix or suffix to the viewing group: "53040.OEM".
    * The mariner should never see these values; they should be shown the name of the viewing group: "Mariners Notes".

The registry can already provide a list of VG's/VGL's, so I'm not sure what added value is provided by listing them in Annex C. Removing the info from Annex C might also encourage more focus on updating the registered information:
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/7bf1625a-10d2-45cd-8985-dec629136b09)

**Comment by alvarosanuy** [comment]
**Decisions made at Portrayal subWG meeting on 17/10/23**

S-98 Change proposal submitted to S100WG8 (November 2023) by NIWC:
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/70189016/020ebf94-77e5-498c-9b88-fd4ccd548de7)

- [x] **NIWC** to confirm approval by S100WG8
- [ ] **Alvaro** to register VG in GI Registry when endorsed by S100WG8

**Comment by DavidGrant-NIWC** [comment]
This was not approved by WG8, but it wasn't rejected either. Since the tables in S-98 will be marked informative (https://github.com/iho-ohi/98-interoperability/issues/19) IMO we are free to proceed as we see fit. I entered an issue in the S-98 repo to cover the action from WG8: https://github.com/iho-ohi/98-interoperability/issues/33

The current contents of the PC match the proposal:

![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/5cd1527d-d390-48c2-b0db-72ffd4bccf61)

---
### WG8 Actions
[WG8 proposal](http://iho-portal.bluemap.kr/share/files/360)
![image](https://github.com/S-101-Portrayal-subWG/Working-Documents/assets/62906211/1d99563c-50a3-4177-b533-d2674853151f)

**Comment by alvarosanuy** [comment]
**Implemented in PC 1.2.0**

- [ ] **Alvaro** to register VG in GI Registry
- [ ] @kusala9 to confirm S-98 and S-164 are in line with changes to Pc 1.2.0

**Comment by DavidGrant-NIWC** [comment]
Note: these viewing groups will likely be expanded due to the new requirements in MSC.530(106)

**Comment by alvarosanuy** [comment]
**Decisions made at Portrayal subWG meeting on 09/04/24**

Put on hold the following open actions awaiting resolution of current discussions with ENCWG on practical implementation of the requirements listed in IMO's MSC.530(106)

> * [ ]  **Alvaro** to register VG in GI Registry
> * [ ]  @kusala9 to confirm S-98 and S-164 are in line with changes to Pc 1.2.0

This issue has dependencies with https://github.com/S-101-Portrayal-subWG/Working-Documents/issues/159
In [this comment](https://github.com/S-101-Portrayal-subWG/Working-Documents/issues/159#issuecomment-2046567051) you can find the latest version of the A&I mapping table and current mappings to 'Alert Highlight'' VGs.

**Comment by alvarosanuy** [comment]
**Decision at PsWG meeting on 10/7/2024**

**Close this issue** and ensure that the IHO's Infrastructure group are aware that all VG & VGL in the GI Registry need to be checked against PC 2.0.0 once approved and remediated when needed (some are missing registration, others are registered but with no metadata populated, etc).
Refer to https://github.com/iho-ohi/S100Infrastructure/issues/20
